### ISSUE TITLE
Inconsistencies with addresses and highlighted dots on map

### ISSUE THREAD

**Issue by KatieRussellTBE** [InFact-coop/building-exploratory#60]
Some of the addresses and the highlighted dots do not match up.

Volunteer comments: 
The location dots on the map need to be numbered with the same numeration on the left hand photos.

The building on left side and the map appeared to show where the property is - is the coloured circle marking the location of the building being highlighted ? but do not seem to be accurate. Working the other way from the map - the highlighting seemed to bring up the right left hand detailed information correctly

**Comment by KatieRussellTBE** [comment]
I've tested the website on my mobile phone, an iphone 7, and the map and dots - highlighting and moving between details/map/thumbnails seems to have a lot of glitches. 
Often when you click on the map and a dot highlights, once you go to the details page and then back to the map, the dot is no longer highlighted and in addition, other dots you click on will not highlight.

**Comment by jessicasalmon** [comment]
hey @KatieRussellTBE. 

> Often when you click on the map and a dot highlights, once you go to the details page and then back to the map, the dot is no longer highlighted

This isn't a bug, this is just how this feature works. When a user clicks on a map marker, it highlights and shows details about that building. Thus, the expected behavior when closing a details view is to close the details page and deselect that marker. 

> in addition, other dots you click on will not highlight.

We'll look into this and report back

**Comment by jessicasalmon** [comment]
hey @KatieRussellTBE we've had a look into it. None of the dots highlight on mobile because as soon as you click a marker, you're taken the details page which takes up the full screen thus the coloured-in marker is hidden behind that. This is the layout and not a bug. The reason you can see the marker change colour on desktop is because the details view takes up only half the screen, thus the map and markers remain visible.

**Comment by KatieRussellTBE** [comment]
@jessicasalmon Thanks for this. What you've described seems to be happening now on my phone, but yesterday it was definitely highlighting the dot on the map first. I then had to click the dot again to get through to the details page, and then when I went back, the dot was no longer highlighted. The map was often loading only in part too, so perhaps it was having a bad day? 
This seems to function fine on the desktop.
